src/tasks/exportEdges.js

```diff
@@ -3,35 +3,27 @@ import { performance } from 'perf_hooks';
 
 import logger from '../helpers/logger';
 import processor from './processor';
-import reduceCapacities from '../helpers/reduce';
-import { getStoredEdges } from '../services/edgesDatabase';
+
 import { redisUrl, redisOptions } from '../services/redis';
 import { writeToFile } from '../services/edgesFile';
 
-const exportEdges = new Queue('Export edges to json file', redisUrl, {
+const exportEdges = new Queue('Export edges to csv', redisUrl, {
   settings: redisOptions,
 });
 
 processor(exportEdges).process(async () => {
   // Measure time of the whole process
   const startTime = performance.now();
 
-  // Get edges from database and write them to the .json file
-  const edges = await getStoredEdges({ hasOnlyFileFields: true });
-  // @FIXME(adz): We filter out too small edges and add a small negative buffer
-  // to the others, to work around small inaccuracies in our database with the
-  // actual trust limits on the blockchain. We plan to remove this when the
-  // trust indexing algorithm got stabilized.
-  const filteredEdges = reduceCapacities(edges);
-  await writeToFile(filteredEdges);
+  // Write edges.csv
+  await writeToFile();
 
-  // Show metrics
+  // End time
   const endTime = performance.now();
   const milliseconds = Math.round(endTime - startTime);
 
-  logger.info(
-    `Written ${filteredEdges.length} of ${edges.length} edges to file in ${milliseconds}ms`,
-  );
+  // Show metrics
+  logger.info(`Written edges.csv in ${milliseconds}ms`);
```

Comment:
I would still add the info of how many edges we've written

Reply:
I would have to open the csv file and coun the number of rows - not sure if we want to do ? maybe the WriteFlile() can return the number of rows?

Reply:
🤔 right... Do we really want to log the amount of edges?
We could do it with the gnu/linux command: `wc -l <filename>`. And it could provide useful insight in the logs

Reply:
Done :) !!